frontendFile/src/main/java/com/virtuslab/gitmachete/frontend/file/highlighting/MacheteAnnotator.java

```diff
@@ -66,6 +69,14 @@ private void processIndentationElement(PsiElement element, AnnotationHolder hold
 
     IndentationParameters indentationParameters = findIndentationParameters(element);
 
+    // Potentially unrelated - it's responsible to change default TAB key behavior
```

Comment:
Hmm just curious if this piece can be executed in a different method? Since `processIndentationElement` looks like sth that's invoked multiple times during the edit, and this piece looks like sth that just needs to be invoked once-ish... but if there's no other way, pls leave a comment about that in the code

Reply:
Also, `s/to change/of changing`

Reply:
Oh ok, `Maybe this is not the best place for that code but I can't find any better place :/`... yeah, so pls leave a comment in the code then

Reply:
Hmmm, I just thinking loud, maybe open machete file action is better place... to be check if its possible

Reply:
Hmm but pls consider that the tab doesn't need to be opened via the action, it can as well be opened directly from VCS tool window

Reply:
Ok, so for now comment leaved